www/docs/admin/policies.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+include_once '../../includes/easyparliament/init.php';
+#include_once INCLUDESPATH . 'easyparliament/commentreportlist.php';
+#include_once INCLUDESPATH . 'easyparliament/searchengine.php';
+include_once INCLUDESPATH . 'easyparliament/member.php';
+#include_once INCLUDESPATH . 'easyparliament/people.php';
+
+$this_page = 'admin_policies';
```

Comment:
I think this page needs to be present in metadata.php with a parent of admin, otherwise no admin protection will occur on this page.